lib/config/fileconf.go

```diff
@@ -1380,6 +1380,8 @@ type Proxy struct {
 	TunAddr string `yaml:"tunnel_listen_addr,omitempty"`
 	// PeerAddr is the address this proxy will be dialed at by its peers.
 	PeerAddr string `yaml:"peer_listen_addr,omitempty"`
+	// PeerPublicAddr is the address this proxy will be dialed at by its peers.
+	PeerPublicAddr apiutils.Strings `yaml:"peer_listen_public_addr,omitempty"`
```

Comment:
```suggestion
	PeerPublicAddr apiutils.Strings `yaml:"peer_public_addr,omitempty"`
```

To be consistent with how other public_addr fields are named.

Reply:
+1